src/Chainweb/CutDB.hs

```diff
@@ -195,16 +195,16 @@ defaultCutDbParams v ft = CutDbParams
 --    intermediate steps instead of starting all over again.
 --
 -- For the latter to work it is important that the local node only pulls cuts
--- that are at most 'forAheadThreshold' blocks in the ahead. Otherwise the
--- pulled blocks would be immediately rejected by the cut processing pipeline
+-- that are at most 'forAheadThreshold' blocks ahead. Otherwise the pulled
+-- blocks would be immediately rejected by the cut processing pipeline
 -- 'processCuts' below in the module and the node would never be able to join
 -- the consensus of the network.
 --
 -- NOTE: THIS NUMBER MUST BE STRICTLY LARGER THAN THE RESPECTIVE LIMIT IN
 -- 'CutDB.Sync'
 --
-farAheadThreshold :: Int
-farAheadThreshold = 2000
+farAheadThreshold :: BlockHeight
```

Comment:
The "MUST BE STRICTLY LARGER THAN" comment is now confusing (and was already a little vague by not explicitly mentioning `catchupStepSize`) as 200 < 1000. Of course 200 is now 2000 or 4000 so it's still true but please update the comment.